.stylelintrc.json

```diff
@@ -9,7 +9,7 @@
   "rules": {
     "a11y/media-prefers-reduced-motion": true,
     "a11y/no-outline-none": true,
-    "a11y/selector-pseudo-class-focus": true,
+    "a11y/selector-pseudo-class-focus": null,
```

Comment:
I think you can just remove this line to disable. IIRC the way the a11y plugin works is that you have to explicitly enable rules.